draft-ietf-rtcweb-jsep.xml

```diff
@@ -1097,6 +1097,54 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           simulcast as described in
           <xref target="sec.simulcast" />).</t>
         </section>
+
+        <section title="stopTransceiver" anchor="sec.transceiver-stop">
+          <t>
+            The stopTransceiver method stops an RtpTransceiver.
+            See <xref target="sec.transceiver-stopped"/> for more details.
+          </t>
+        </section>
+        <section title="isTransceiverStopped" anchor="sec.transceiver-stopped">
+          <t>
+            The isTransceiverStopped method returns true if
+            the transceiver has been stopped, either by a call
+            to stopTransceiver or by a call to
+            setRemoteDescription, and false otherwise.
+          </t>
+          <t>
+            A stopped RtpTransceiver does not send any
+            outgoing RTP or RTCP or process any incoming RTP
+            or RTCP.
+          </t>
+        </section>
+        <section title="setTransceiverDirection" anchor="sec.transceiver-set-direction">
+          <t>
+            The setTransceiverDirection method sets the
+            direction of a transceiver, which affects the
+            direction attribute of an associted m=section on
+            future calls to createOffer and createAnswer.
+          </t>
+        </section>
+        <section title="setCodecPreferences" anchor="sec.transceiver-set-codec-preferences">
+          <t>
+            The setCodecPreferences method sets the codec
+            preferences of a transceiver, which affect the
+            presence and order of codecs of an associated
+            m=section on future calls to createOffer and
+            createAnswer.
+          </t>
+          <t>
+            The codec preferences of an RtpTransceiver can
+            affect calls to createOffer and createAnswer by
```

Comment:
Then how does an answerer specify which codec the prefer to receive?  The W3C spec specifically says "This method allows applications to disable the negotiation of specific codecs. It also allows an application to cause a remote peer to prefer the codec that appears first in the list for sending."  Would that not apply to both the offerer and the answerer?